quaiclient/ethclient/ethclient.go

```diff
@@ -258,19 +258,6 @@ func (ec *Client) TransactionInBlock(ctx context.Context, blockHash common.Hash,
 	return json.tx, err
 }
 
-// TransactionReceipt returns the receipt of a transaction by transaction hash.
-// Note that the receipt is not available for pending transactions.
-func (ec *Client) TransactionReceipt(ctx context.Context, txHash common.Hash) (*types.Receipt, error) {
```

Comment:
@jdowning100 uses this in the test, we should keep it. When we merge the ethclient and quaiclient.